components/TopBar/CategoryDescriptions.ts

```diff
@@ -94,6 +94,8 @@ const categoryDescriptions: CategoryDescriptions = {
     'Anyone can contribute to the projects are called OpenSource Projects. There are many open source projects available for developers. Some of the most popular ones include Linux, Git, and Apache1. Other open source projects include Kubernetes, TensorFlow, and React Native',
   tools:
     'There are various open source tools available for developers. Some of the most popular ones include Hadoop, VLC Media Player, and GIMP1. Other open source tools include TensorFlow, Eclipse Che, and OpenShift Do.',
+  'os tutorials':
+    'OS tutorials helps you to learn-by-watching. It contains the list most helping tutorials on open-source.',
```

Comment:
Why have you added this?

Reply:
@hritikbhattacharya, This simply isn't required. We have renamed the `os tutorials` as `tutorials`, and its description has already been added [here](https://github.com/rupali-codes/LinksHub/pull/2180/files#diff-6255e5230971ba71d0f0818df0f73b28825755e29ede66a7336941cc5a5e4c80R99-R100).

Kindly, please revert this change.

```suggestion
```

Reply:
removed